app/src/net/fieldwire/SampleCalls.java

```diff
@@ -0,0 +1,280 @@
+                .client(this.okHttpClient)
+                .build()
+                .create(SuperEndpoint.class);
+
+        List<AccountUserResponse> users = tokenManager.executeWithToken(endpoint::getAccountUsers);
+        String users_info = users
+                .stream()
+                .map(u -> String.format("%s: %s", u.user().accountRole, u.user().email))
+                .sorted()
+                .collect(Collectors.joining("\n"));
+
+        log_info("Users in account", users_info);
+    }
+
+    // Regional endpoints are used for projects, templates etc.
+    private void executeSampleRegionalCalls(
+            @NotNull Region region,
+            @NotNull String emailForProjectInvite,
+            @NotNull String localFilePathForPlan
+    ) throws IOException {
+        // Based on the region, figure out which base URL we want to talk with
+        final String regionalBaseUrl;
+        if (region == Region.EU) {
+            regionalBaseUrl = RegionalEndpoint.API_BASE_URL_EU;
+        } else if (region == Region.US) {
+            regionalBaseUrl = RegionalEndpoint.API_BASE_URL_US;
+        } else {
+            throw new RuntimeException("Unrecognizable region: " + region);
+        }
+
+        // Build the impl that allows super endpoints to be called
+        RegionalEndpoint endpoint = new Retrofit.Builder()
+                .baseUrl(regionalBaseUrl)
+                .addConverterFactory(GsonConverterFactory.create(gson))
+                .client(this.okHttpClient)
+                .build()
+                .create(RegionalEndpoint.class);
+
+        //-----------------------------------------------
+        // List projects
+        //-----------------------------------------------
+
+        List<Project> projects = tokenManager.executeWithToken(endpoint::getProjects);
+        String projects_info = projects
+                .stream()
+                .map(p -> String.format("%s: %s", p.id, p.name))
+                .sorted()
+                .collect(Collectors.joining("\n"));
+
+        log_info("Projects in account", projects_info);
+
+        //-----------------------------------------------
+        // Create a new project
+        //-----------------------------------------------
+
+        Project project = tokenManager.executeWithToken(authz -> endpoint.createProject(
+                authz,
+                new CreateProject("New project")
+        ));
+
+        log_info("Created project", project.toString());
+
+        //-----------------------------------------------
+        // Invite a user to a project
+        //-----------------------------------------------
+
+        ProjectUsersResponse projectUsersResponse = tokenManager.executeWithToken(
+                authz -> endpoint.inviteUser(
+                    authz,
+                    project.id,
+                    new InviteUser(emailForProjectInvite)
+                )
+        );
+
+        ProjectUser user = projectUsersResponse.users.get(0);
```

Comment:
nit: could be great handling the case where there is no users in the response (I naturally put the email address of my account owner, which is ok but returns an empty array).

What about:
```
 if (projectUsersResponse.users.stream().count() == 0) {
      throw new RuntimeException("User already exists in the project, use a different email address " + emailForProjectInvite);
 }
```

Reply:
Good point! Addressed - https://github.com/Fieldwire/fieldwire_java_sample/pull/3/commits/6797ff1135c65d7cd21080eeb2fc18c73ac14247